README.md

```diff
@@ -1,48 +1,54 @@
 
+Stellarphot provides a Python package and accompanying Jupyter notebooks to allow you to turn reduced astronomical images of point sources (e.g. stars) into calibrated astronomical photometry, with a focus on variable star and exoplanet transit observations.  Specifically:
 
-+ If you already have aperture photometry for a field,
-    * calculate relative flux (like [AstroImageJ](https://www.astro.louisville.edu/software/astroimagej/)), and/or
-    * calculate calibrated magnitudes by transforming to a catalog (e.g. APASS DR9)
-+ If you have calibrated images but haven't done photometry yet, you can do aperture photometry on your images.
-+ If you have not calibrated your images, `stellarphot` can help -- choose the
-
++ If you have reduced astronomical images as FITS files but haven't obtained photometry yet, `stellarphot` can perform aperture photometry on your images.
++ If you already have aperture photometry for a field, `stellarphot` can 
+  * calculate relative flux (like [AstroImageJ](https://www.astro.louisville.edu/software/astroimagej/)), and/or
+  * calculate calibrated magnitudes by transforming to a catalog (e.g. APASS DR9)
++ If you are working with exoplanet transit observations, `stellarphot` can turns the photometry into exoplanet transit light curves (see installation notes below).
 
 ## Installation
 
-You can install `stellarphot` with either pip or conda. `conda` is recommended at the moment because of an issue with installing one of the dependencies of `stellarphot` in Python 3.9 or higher
+You can install `stellarphot` with either `pip` or `conda`.  If you are interested in `stellarphot` for exoplanet transit light curves, `conda` is recommended at the moment because of an issue with installing one of the dependencies.
+
++ Install with `conda`:
+
+  	conda install -c conda-forge stellarphot reducer
+  	pip install astronbs
+
+  * If you are interested in exoplanet light curve fitting, also install `batman` via 
 
-Install with `conda`:
+    	conda install -c conda-forge batman-package
 
-```
-conda install -c conda-forge stellarphot reducer
-pip install astronbs
-```
++ Install with `pip`:
 
-or install with `pip`:
+  	pip install stellarphot reducer astronbs
 
-```
-pip install stellarphot reducer astronbs
-```
+  * If you are interested in exoplanet light curve fitting, also  install `batman` via:
 
-## Running stellarphot
+    	pip install batman-package
```

Comment:
they could also get this by doing `pip install stellarphot[exo_fitting]`

Reply:
Fixed, since I think this is cleaner.